packages/sysutils/busybox/package.mk

```diff
@@ -156,7 +156,7 @@ makeinstall_target() {
     cp $PKG_DIR/scripts/createlog $INSTALL/usr/bin/
     cp $PKG_DIR/scripts/lsb_release $INSTALL/usr/bin/
     cp $PKG_DIR/scripts/apt-get $INSTALL/usr/bin/
-    cp $PKG_DIR/scripts/passwd $INSTALL/usr/bin/
+    #cp $PKG_DIR/scripts/passwd $INSTALL/usr/bin/
```

Comment:
should be deleted instead of commented out

Reply:
done